README.md

```diff
@@ -1 +1,183 @@
-# Helo
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
Please consider either removing this section or replacing it with your own logo only if you have one as its a project requirement to have a professional repository. Moreover, if you want to give credit to Microverse rather than writing here `Microverse README Template` you can use the Acknowledgments section.